src/rules/typedefRule.ts

```diff
@@ -155,3 +181,11 @@ class TypedefWalker extends Lint.RuleWalker {
         }
     }
 }
+
+function isPropertyDeclaration(node: ts.Node): boolean {
+    return node.kind === ts.SyntaxKind.PropertyDeclaration;
+}
+
+function isTypedPropertyDeclaration(node: ts.Node): boolean {
+    return isPropertyDeclaration(node) && !!(node as ts.PropertyDeclaration).type;
```

Comment:
Then with the typeguard from above, you can simplify this!

```
return isPropertyDeclaration(node) && node.type != null;
```

(The TSLint code-style uses explicit comparisons to null instead of `!!` to avoid the weirdness that happens with JS implicit type conversion. For example, `!!"" !== !!"0"`)